### DISCUSSION TITLE
[ISSUE] Mouse follower - onScroll

### DISCUSSION THREAD

**Discussion by ilyasseisov** [motiondivision/motion#2859] [Q&A]
Hi!

Could you please help?
I have a mouse follower effect, it works, but I need it to **stick to the cursor when scroll happens**.
I've tried many things, but no success.

Example of the website where it works as I need:
https://minhpham.design/

Here is my StackBlitz:
https://stackblitz.com/edit/vite-react-tailwind-qeksd3?file=src%2FApp.jsx

Thanks in advance!

**Comment by mattgperry** [comment]
Is it at all possible to add a scroll handler into useMousePosition that adds the scroll offset to the latest x/y position?

**Reply by ilyasseisov** [reply]
Hi!

I've done that. 
That results in mouse follower be following the cursor. Not stick to it.
